libhttpserver/server.go

```diff
@@ -112,13 +131,16 @@ func (s *Server) getHTTPFileSystem(ctx context.Context, requestPath string) (
 // For example:
 //     /team/keybase/file.txt?token=1234567890abcdef1234567890abcdef
 func (s *Server) serve(w http.ResponseWriter, req *http.Request) {
+	s.logger.Info("incoming request from %q: %s", req.UserAgent(), req.URL)
```

Comment:
Also usually we try to capitalize the first letter of log messages, here and below.